lib/Fhp/Action/GetBalance.php

```diff
@@ -48,21 +48,41 @@ public static function create(SEPAAccount $account, bool $allAccounts = false):
         return $result;
     }
 
+    /**
+     * @deprecated Beginning from PHP7.4 __unserialize is used, then this method is never called
+     */
     public function serialize(): string
     {
-        return serialize([
-            parent::serialize(),
+        return serialize($this->__serialize());
+    }
+
+    public function __serialize(): array
+    {
+        return [
+            parent::__serialize(),
             $this->account, $this->allAccounts,
-        ]);
+        ];
     }
 
+    /**
+     * @deprecated Beginning from PHP7.4 __unserialize is used, then this method is never called
+     *
+     * @param string $serialized
+     * @return void
+     */
     public function unserialize($serialized)
+    {
+        $this->__unserialize(unserialize($serialized));
+    }
+
+    public function __unserialize(array $serialized): void
     {
         list(
             $parentSerialized,
             $this->account, $this->allAccounts
-            ) = unserialize($serialized);
-        parent::unserialize($parentSerialized);
+            ) = $serialized;
+
+        parent::__unserialize($parentSerialized);
```

Comment:
My comments on this file apply analogously to all other similar files in this commit.